src/main.rs

```diff
@@ -1,234 +1,18 @@
-use anyhow::{bail, Result};
-use clap::Parser;
-use kanata_parser::cfg;
-use kanata_state_machine::*;
-use log::info;
-use simplelog::{format_description, *};
-use std::path::PathBuf;
+#![cfg_attr(feature = "gui", windows_subsystem = "windows")] //disable console on Windows
 
-#[cfg(test)]
-mod tests;
+#[cfg(not(feature = "gui"))]
+use kanata_state_machine::lib_main::lib_main_cli;
+#[cfg(feature = "gui")]
+use kanata_state_machine::lib_main::lib_main_gui;
 
-#[derive(Parser, Debug)]
-#[command(author, version, verbatim_doc_comment)]
```

Comment:
Please undo this refactor to move code to a new file.

It can be done in a later PR but I don't want to have to review such a refactor in this PR.

Reply:
moved, though that means even more dupes, the whole Args struct